[DE] Add `Beatmap/Beatmap_collaborations` (#14994)
ppy/osu-wiki: wiki/Beatmap/Beatmap_collaborations/de.md

---
tags:
  - collab
  - collabs
---

# Beatmap-Kollaborationen

**Beatmap-Kollaborationen**, kurz ***collabs***, sind einzelne [Schwierigkeitsgrade](/wiki/Beatmap/Difficulty) oder [Beatmaps](/wiki/Beatmap) mit [Gastschwierigkeiten](/wiki/Beatmap/Guest_difficulty), die von mehreren Personen erstellt wurden. Während sich der Begriff *Kollaboration* meistens auf Beatmaps bezieht, kann er auch für andere Medien, die von mehreren Teilnehmern erstellt wurden, verwendet werden, beispielsweise [Storyboards](/wiki/Storyboard) oder Songs.

## Beschreibung

> Normalerweise sind das Wort "Collab" oder mehrere Benutzernamen im Namen des Schwierigkeitsgrades oder in der [Beatmapbeschreibung](/wiki/Beatmap/Beatmap_description) einer Beatmap-Kollaborationen enthalten, obwohl es keine strikte Anforderung der [Ranking-Kriterien](/wiki/Ranking_criteria) ist. Es ist auch üblich, dass Mapper, die häufig kollaborieren, Verschmelzungen ihrer Benutzernamen erfinden[^name-example] und diese im Verlauf ihrer Mapping-Karrieren konsequent verwenden.

Reviewer: ```suggestion
Normalerweise sind das Wort "Collab" oder mehrere Benutzernamen im Namen des Schwierigkeitsgrades oder in der [Beatmapbeschreibung](/wiki/Beatmap/Beatmap_description) einer Beatmap-Kollaboration enthalten, obwohl es keine strikte Anforderung der [Ranking-Kriterien](/wiki/Ranking_criteria) ist. Es ist auch üblich, dass Mapper, die häufig kollaborieren, Verschmelzungen ihrer Benutzernamen erfinden[^name-example] und diese im Verlauf ihrer Mapping-Karrieren konsequent verwenden.
```

> Der Mapping-Prozess beginnt normalerweise damit, dass der [Besitzer der Beatmap](/wiki/Beatmap/Beatmap_host) eine Beatmap mit [Timing](/wiki/Beatmapping/Timing) entsprechend der musikalischen Struktur des Songs in mehrere Abschnitte einteilt, indem er diese mithilfe von Bookmarks auf der [Timeline](/wiki/Client/Beatmap_editor/Timelines) markiert. Relativ lange Abschnitte, wie etwa Strophen, werden üblicherweise in Segmente von je vier oder acht [Takten](/wiki/Music_theory/Measure) eingeteilt. Die Teilnehmer einigen sich anschließend auf eine Aufteilung (im Fall von zwei Mappern z. B. nach geraden und ungeraden Segmenten) und allen wird ein Abschnitt zugewiesen.

Reviewer: ```suggestion
Der Mapping-Prozess beginnt normalerweise damit, dass der [Besitzer der Beatmap](/wiki/Beatmap/Beatmap_host) eine Beatmap mit [Timing](/wiki/Beatmapping/Timing) entsprechend der musikalischen Struktur des Songs in mehrere Abschnitte einteilt, indem er diese mithilfe von Lesezeichen auf der [Zeitleiste](/wiki/Client/Beatmap_editor/Timelines) markiert. Relativ lange Abschnitte, wie etwa Strophen, werden üblicherweise in Segmente von je vier oder acht [Takten](/wiki/Music_theory/Measure) eingeteilt. Die Teilnehmer einigen sich anschließend auf eine Aufteilung (im Fall von zwei Mappern z. B. nach geraden und ungeraden Segmenten) und allen wird ein Abschnitt zugewiesen.
```

> Alle Mapper vollenden ihren Teil, indem sie die [Hit-Objekte](/wiki/Gameplay/Hit_object) platzieren, [Hitsounds](/wiki/Beatmapping/Hitsound) hinzufügen, bei Bedarf [nicht vererbte Timing-Punkte](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) verwenden und schließlich ihr Werk an den Besitzer der Beatmap senden, der daraufhin alle Änderungen in einem einzelnen Schwierigkeitsgrad kombiniert, indem er sorgfältig die zugehörige [`.osu`-Datei](/wiki/Client/File_formats/osu_(file_format)) bearbeitet.

Reviewer: nicht vererbte Timing-Punkte? Der originale Artikel verweist auf vererbte Timing-Punkte.

Author: Danke. Ich frag mich, wieso ich den Link nicht einfach kopiert hab

> Um zu erzielen, dass alle Übergänge im Spielverlauf möglichst reibungslos sind, verlassen sich die Mapper in aller Regel auf die bereits fertiggestellten Abschnitte. Dennoch existieren sogenannte *blinde Kollaborationen* (*blind collabs*), wie beispielsweise [Mitose Noriko - Unreal Player](https://osu.ppy.sh/beatmapsets/828563#osu/1735958) von ::{ flag=DE }:: [Mao](https://osu.ppy.sh/users/2204515), bei denen die Mapper die Abschnitte der Anderen nicht zu sehen bekommen bis die Beatmap fertiggestellt ist. In solchen Fällen werden am Ende jedes Abschnitts typischerweise vorab Hit-Objekte platziert, um zu kennzeichnen an welcher Stelle die Beatmap fortgesetzt werden soll.

Reviewer: ```suggestion
Damit alle Übergänge im Spielverlauf möglichst reibungslos sind, verlassen sich die Mapper in aller Regel auf die bereits fertiggestellten Abschnitte. Dennoch existieren sogenannte *blinde Kollaborationen* (*blind collabs*), wie beispielsweise [Mitose Noriko - Unreal Player](https://osu.ppy.sh/beatmapsets/828563#osu/1735958) von ::{ flag=DE }:: [Mao](https://osu.ppy.sh/users/2204515), bei denen die Mapper die Abschnitte der Anderen nicht zu sehen bekommen, bis die Beatmap fertiggestellt ist. In solchen Fällen werden am Ende jedes Abschnitts typischerweise vorab Hit-Objekte platziert, um zu kennzeichnen an welcher Stelle die Beatmap fortgesetzt werden soll.
```
